kubernetes/tekton-resources/demo/pipeline-run.yaml

```diff
@@ -17,5 +17,5 @@ spec:
       claimName: kaniko-source-pvc
   - name: docker-secrets
     secret:
-      secretName: secret-dockerconfigjson
+      secretName: regcred
```

Comment:
Doing it this way will fail. Regcred in this namespace is set to .dockerconfigjson and the kaniko task needs to have an opaque secret that is a config.json since the kaniko task doesn't correctly use the .dockerconfigjson model.

Reply:
There must be something broken in one of the scripts upstream then - this is not working with current config for me.  Will dig a little deeper